### PR TITLE
feat: Add unit-tests for @PrimaryKey and @FirstKey/@SequenceKey

### PR DESCRIPTION
**Pull request by bdferris-v2**
In preparation for multi-colum primary key refactoring to come in support of #1187, this PR adds unit-tests for @PrimaryKey and @FirstKey/@SequenceKey

- [X] Run the unit tests with `gradle test` to make sure you didn't break anything
- [X] Format the title like "feat: [new feature short description]". Title must follow the Conventional Commit Specification(https://www.conventionalcommits.org/en/v1.0.0/).
- [X] Linked all relevant issues

### CODE REVIEW

File: core/src/main/java/org/mobilitydata/gtfsvalidator/testing/LoadingHelper.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+  private NoticeContainer noticeContainer = new NoticeContainer();
+
+  public List<ValidationNotice> getValidationNotices() {
+    return noticeContainer.getValidationNotices();
+  }
+
+  public <X extends GtfsEntity, Y extends GtfsTableContainer<X>> Y load(
+      GtfsTableLoader<X> loader, String... lines) throws ValidatorLoaderException {
+    String content = Arrays.asList(lines).stream().collect(Collectors.joining("\n"));
```
Comment (asvechnikov2) [inline core/src/main/java/org/mobilitydata/gtfsvalidator/testing/LoadingHelper.java:54]: nit: It should be possible to use Arrays.stream(lines) directly, even stream(lines) with static import.

File: core/src/main/java/org/mobilitydata/gtfsvalidator/testing/LoadingHelper.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    InputStream in = new ByteArrayInputStream(content.getBytes(StandardCharsets.UTF_8));
+
+    ValidationContext context =
+        ValidationContext.builder()
+            .setCountryCode(countryCode)
+            .setCurrentDateTime(new CurrentDateTime(currentTime))
+            .build();
+    ValidatorLoader validatorLoader = new ValidatorLoader(ImmutableList.of());
```
Comment (asvechnikov2) [inline core/src/main/java/org/mobilitydata/gtfsvalidator/testing/LoadingHelper.java:62]: Could you please add a comment as to why we want to pass an empty list of validatorPackages?

**Review by asvechnikov2 — APPROVED** [review]
LGTM! Thanks!

File: core/src/main/java/org/mobilitydata/gtfsvalidator/testing/LoadingHelper.java (hunk 2 above)
Comment (bdferris-v2) [reply core/src/main/java/org/mobilitydata/gtfsvalidator/testing/LoadingHelper.java:66]: Done.

File: core/src/main/java/org/mobilitydata/gtfsvalidator/testing/LoadingHelper.java (hunk 1 above)
Comment (bdferris-v2) [reply core/src/main/java/org/mobilitydata/gtfsvalidator/testing/LoadingHelper.java]: Done.  Thanks for the tip!

**Comment by bdferris-v2** [comment]
Hey @maximearmstrong, Artur has given this an LGTM but due to some issues with his work computer + GitHub account, he had to reset his account and lost review privileges.  Any chance you can give this an approval while that's being sorted out?

**Review by maximearmstrong — APPROVED** [review]
LGTM! Thank you @bdferris-v2

**Pull request merged**